Renesas RZ_A1H missing NVIC_SystemReset

### Description
- Type: Bug or Enhancement | Question
- Related issue: 
- Priority:  Major 

---------------------------------------------------------------
## Bug

**Target**
 RZ_A1H

**Toolchain:**
GCC_ARM|ARM|IAR

**Toolchain version:**
N/A

**mbed-cli version:** 
(`mbed --version`)
N/A

**mbed-os sha:**
(`git log -n1 --oneline`)
C:\Users\jankii01\mbed\mbed-os-example-client\mbed-os>git log -n1 --oneline
c59c40009 (HEAD -> master, origin/master, origin/HEAD) Merge pull request #5832 from geky/fix-truncate-zero

**DAPLink version:**
N/A

**Expected behavior**

mbed-os-example-client compileshttps://connector.mbed.com/

**Actual behavior**

Doesn't compile, due to missing  reboot definition.

```
  ^~~~~~~
.\pal\Source\Port\Reference-Impl\mbedOS\RTOS\pal_plat_rtos.cpp: In function 'void pal_plat_osReboot()':
.\pal\Source\Port\Reference-Impl\mbedOS\RTOS\pal_plat_rtos.cpp:243:22: error: 'NVIC_SystemReset' was not declared in this scope
     NVIC_SystemReset();
```


**Steps to reproduce**
```
git clone git@github.com:ARMmbed/mbed-os-example-client.git
- get Mbed Connect security.h -file (https://connector.mbed.com/)
mbed compile -m RZ_A1H  -t <your toolchain>
```

This will be needed for Mbed Cloud Client -enablement.

--- a/targets/TARGET_RENESAS/TARGET_RZ_A1XX/TARGET_RZ_A1H/device/cmsis.h
+++ b/targets/TARGET_RENESAS/TARGET_RZ_A1XX/TARGET_RZ_A1H/device/cmsis.h
@@ -8,5 +8,6 @@
 #define MBED_CMSIS_H
 
 #include "MBRZA1H.h"
+#include "cmsis_nvic.h"
 
 #endif

